src/resources/lang/de/crud.php

```diff
@@ -20,7 +20,7 @@
     'save_action_changed_notification' => 'Standardverhalten nach dem Speichern wurde geändert.',
 
     // Create form
-    'add'                 => 'Neu',
+    'add'                 => 'Anlegen: ',
```

Comment:
`Neu [thing]` is just grammatically wrong. German nouns can have one of three genders assigned and `Neu` matches none of those (which is fine because it may be better to hit none than just be right sometimes).
`Anlegen: [thing]` is not beautiful, but it's grammatically neutral and therefore correct.